main.go

```diff
@@ -181,3 +205,75 @@ func httpErrorHandler(err error, c echo.Context) {
 func requestsStatusEndpoint(context echo.Context) bool {
 	return context.Request().RequestURI == "/status"
 }
+
+// createTokenGenerator generates valid API tokens for the Nuts node and signs them with the private key
+func createTokenGenerator(config Config) core.AuthorizationTokenGenerator {
+	return func() (string, error) {
+		key, err := jwkKey(config.apiKey)
+		if err != nil {
+			return "", err
+		}
+
+		issuedAt := time.Now()
+		notBefore := issuedAt
+		expires := notBefore.Add(time.Second * time.Duration(5))
+		token, err := jwt.NewBuilder().
+			Issuer(config.NutsNodeAPIUser).
+			Subject(config.Credentials.Username).
+			Audience([]string{config.NutsNodeAddress}).
```

Comment:
shouldn't this be the configured name? Or at least just the hostname from the address

Reply:
`aud` is specified to be a URI. The configured address is a URI. The node should have the correct value configured. Clients should use the address.